_faq/general/create-issue.md

```diff
@@ -0,0 +1,11 @@
+---
+title: I have an idea/found a bug. What do I do next?
+order: 2
+---
+{::options auto_ids="false" /}
```

Comment:
Help me understand why we're disabling these?

Reply:
No idea what those are.